lstchain/calib/camera/flatfield.py

```diff
@@ -43,6 +43,11 @@ class FlasherFlatFieldCalculator(FlatFieldCalculator):
         [-3, 3],
         help='Interval (number of std) of accepted charge standard deviation around camera median value'
     ).tag(config=True)
+    time_calibration_path = Unicode(
+        '',
```

Comment:
The `allow_none` only makes sense if the default value is also None. I would recommend to use it this way.

So `Unicode(None, allow_none=True, help='...')`

Reply:
I put None as default